login !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!

nose221834 opened the pull request:
<!-- 全部埋める必要はありませんが，できるだけわかりやすく書いてください -->
# 対応Issue
<!-- 対応したIssue番号を記載 -->
resolve #0

# 概要
<!-- 開発内容の概要を記載 -->
- 

# 実装詳細
<!-- 具体的な開発内容を記載 -->
- 

# 画面スクリーンショット等
<!-- URLとともに貼る（なければ空欄でよい） -->


# テスト項目
<!-- テストしてほしい内容を記載 -->
<!-- ex) コンポーネントのデザインが崩れないか -->
<!-- ex) データが表示できてるか・反映されてるか -->
- [ ] 
- [ ] 
- [ ] 

# 備考
<!-- 実装していて困った箇所・質問など -->

nose221834 committed:
feat devise-jwtを追加

nose221834 committed:
run: rails generate devise User

nose221834 committed:
feat: jetでのsessionを保持するための設定を追加

nose221834 committed:
run: pnpm install next-auth

nose221834 committed:
tmp

Kubosaka commented on user/src/api/auth/signup.ts:
```diff
@@ -0,0 +1,45 @@
+// pages/api/auth/signup.ts
+import type { NextApiHandler } from 'next';
+
+const handler: NextApiHandler = async (req, res) => {
```
[ask]これってnext api使わないとできないんかな？

nose221834 committed:
curl -X POST http://localhost:3000/users \
  -H "Content-Type: application/json" \
  -H "Accept: application/json" \
  -d '{
    "user": {
      "email": "newuser@example.com",
      "password": "password123",
      "password_confirmation": "password123",
      "role_id": "4"
    }
  }'

  が実行できる状態

nose221834 committed:
run: rails g model jwt_denylist jti:string exp:datetime
